app/frontend/README.md

```diff
@@ -13,7 +13,7 @@ Communication with the backend is via [protobuf messages](https://github.com/pro
 
 2. Make sure you have the development environment going (see below).
 
-3. Create a new branch for your issue eg. `feature/global-search`, `bug/no-duplicate-users` or `refactor/fix-host-requests`
+3. Create a new branch for your issue under 'frontend/issue-type/branch-name' eg. `frontend/feature/global-search`, `frontend/bug/no-duplicate-users` or `frontend/refactor/fix-host-requests`
```

Comment:
I have been using slashes as 'folders'

I assumed it was common practice because in the git UI that I use (git extensions, I'm too lazy to memorize the commands), it collapses them into nice neat groups

Reply:
I do not know.

Reply:
I've been using slashes (`/`), I like it more, then you can use dashes to form a nice slug. E.g. `backend/feature/cool-new-feature`